libs/database/src/lib/dtos/feed.dto.ts

```diff
@@ -21,39 +21,10 @@
  * or have any questions.
  */
 
-import { TransformStringToArrayOfStrings } from '@castcle-api/utils/commons';
 import { ApiProperty } from '@nestjs/swagger';
 import { Type } from 'class-transformer';
-import {
-  IsArray,
-  IsEnum,
-  IsMongoId,
-  IsNumber,
-  IsOptional,
-  Max
-} from 'class-validator';
-import { DEFAULT_QUERY_OPTIONS } from './common.dto';
-
-export enum UserField {
-  Relationships = 'relationships'
-}
-
-export class ExpansionQuery {
-  @ApiProperty({
-    enum: UserField,
-    required: false,
-    isArray: true
-  })
-  @TransformStringToArrayOfStrings()
-  @IsOptional()
-  @IsArray()
-  @IsEnum(UserField, { each: true })
-  userFields?: UserField[];
-
-  hasRelationshipExpansion = (() => {
-    return Boolean(this.userFields?.includes(UserField.Relationships));
-  }) as unknown as boolean;
-}
+import { IsMongoId, IsNumber, IsOptional, Max } from 'class-validator';
+import { DEFAULT_QUERY_OPTIONS, ExpansionQuery } from './common.dto';
```

Comment:
Merging conflict?
Moving back ExpansionQuery and UserField to `common.dto` can cause circular dependency.